openai/spring-cloud-azure-starter-openai/spring-cloud-azure-openai-sample/src/main/java/com/azure/spring/openai/sample/OpenAIController.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.openai.sample;
+
+import com.azure.ai.openai.OpenAIClient;
+import com.azure.ai.openai.models.Choice;
+import com.azure.ai.openai.models.Completions;
+import com.azure.ai.openai.models.CompletionsOptions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+@RestController
+public class OpenAIController {
+
+    private static final Logger logger = LoggerFactory.getLogger(OpenAIController.class);
+    public static final String deploymentOrModelId = "gpt-35-turbo";
+    @Autowired
+    private OpenAIClient client;
+
+    @PostMapping("/prompt")
+    @ResponseBody
```

Comment:
I don't think the `@ResponseBody` is required here https://www.baeldung.com/spring-controller-vs-restcontroller#overview.